CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [23.10.0](https://github.com/looker-open-source/sdk-codegen/compare/sdk-codegen-all-v23.8.1...sdk-codegen-all-v23.10.0) (2023-06-15)
```

Comment:
Just noticed that 'all' is being calendar versioned. It doesn't get published so it doesn't matter but it shouldn't be CalVer.

Reply:
I think it has to be so that the 23.10.0 version flows to everything else.